src/utils/api.js

```diff
@@ -40,3 +38,9 @@ export const addComment = (articleId, formData) => {
     .post(`/articles/${articleId}/comments`, postBody)
     .then((response) => response.data);
 };
+
+export const deleteComment = (commentId) => {
+  return newsApi
+    .delete(`/comments/${commentId}`)
+    .then((response) => response.data);
```

Comment:
Sould we need to have a `.then(...)` here? What is response.data? Are we using this when calling the function?